src/lib/core/crash.h

```diff
@@ -9,23 +9,124 @@
 extern "C" {
 #endif /* defined(__cplusplus) */
 
-/**
- * PATH_MAX is too big and 2K is recommended
- * limit for web address.
- */
-#define CRASH_FEEDBACK_HOST_MAX 2048
+#if defined(TARGET_OS_LINUX) && defined(__x86_64__)
```

Comment:
You need to include `trivia/config.h`, otherwise `TARGET_OS_LINUX` may not be defined.

Reply:
Fixed.